cs130-w21/18
# Be/start heroku deployment

- Add Heroku specific deployment instructions
  - Procfile - contains command to run server
  - requirements.txt - contains Python dependencies
- Add dummy server code - API can be reached via URL.
- Add dummy tests - test dummy server code
- Add Travis CI support
  - Run tests for all branches that this Travis YML file belongs to (should be all be/* branches)
  - Deploy to backend staging from be/master
  - Deploy to backend prod from be/master on tagged push
